crates/control-interface/src/inv.rs

```diff
@@ -0,0 +1,134 @@
+// Philosophical note: These two main structs (Invocation and InvocationResponse) exist inside
+// the wasmcloud-host crate. If this crate were to refer to that, we'd create a cyclical reference
+// and all perish inside a black hole.
+//
+// The other alternative is to create a new, third crate shared by the two, but that creates a brand
+// new "reason to change" now for three crates and produces a "dependency as a liability"
+//
+// Therefore, it's easier to simply create copies of the data types since we only need them to be
+// able to push them on the wire.
+
```

Comment:
Maybe it would be useful to include a RUSTDOC annotation here with the file that this is a copy of for extra doc info? Something like
```
[Invocation](../wasmcloud-host/struct.Invocation)
```
I'm not sure if that's the correct syntax or not, but I think this would at least provide a reference and help us keep track of the copy

Reply:
It's hard to create an extra-crate link when the crate you're pointing to hasn't been published yet. I'll add some context, though.